src/UI/templates/default/Deck/deck.less

```diff
@@ -21,3 +21,23 @@
     }
   }
 }
+
+@media (min-width: @screen-sm-min) and (max-width: @screen-sm-max) {
+  .row-eq-height {
+
+    display: -webkit-box;
+    display: -webkit-flex;
+    display: -ms-flexbox;
+    display: flex;
+    flex: 3;
+    flex-wrap: wrap;
```

Comment:
Why is this only needed for (min-width: @screen-sm-min) and (max-width: @screen-sm-max) and not just for (min-width: @screen-sm-min)? Others should also wrap, right? Again probably I am missing something.